roles/schema_registry/tasks/get_authorization_token.yml

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Why not parametrise the user name and other variables and reuse the code from common.

Reply:
thanks, updated the playbook with parametrization